docs/docs/creating-and-modifying-pages.md

```diff
@@ -121,7 +121,7 @@ exports.onCreatePage = ({ page, actions }) => {
   return new Promise(resolve => {
     const oldPage = Object.assign({}, page)
     // Remove trailing slash unless page is /
-    page.path = _path => (_path === `/` ? _path : _path.replace(/\/$/, ``))
```

Comment:
If we're going to go this route, why not create a function (e.g. `const replacePath = _path => // yadda yadda`) and then use it like the link you provide? That seems _much_ clearer!

What do you think?